src/match_traildb.c

```diff
@@ -375,11 +375,12 @@ int run_groupby_query2(char **traildb_paths, int num_paths, groupby_info_t *gi,
     j128m_init(states);
 
 
-    __uint128_t *window_cookies = 0;
+    __uint128_t *window_ids = 0;
     uint64_t num_windows = 0;
 
     if (window_set)
-        window_cookies = window_set_get_cookies(window_set, &num_windows);
+        // This is a list of cookies, unless there's a 4th id column, in which case it's a list of ids.
+        window_ids = window_set_get_cookies(window_set, &num_windows);
```

Comment:
I'd rename this function to `window_set_get_ids` or something like that for clarity.

Reply:
Ok. Wasn't sure how much leeway we had in terms of renaming things re: backwards compatibility, but it looks like a lot of these functions are fairly private.